blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -177,6 +178,32 @@ export default async function decorate(block) {
                 </div>
             </div>
         </div>
+        <div id="tab3" class="calendar tab inactive">
+            <div class="control-wrapper">
+                <div class="inc-dec-wrapper">
+                    <div class="year-switch">
+                        <div id="dec-year" class="year-toggle">
+                            <img class="left" data-direction="left" src="/icons/chevron-right.svg"></img>
+                        </div>
+                        <div id="inc-year" class="year-toggle">
+                            <img class="right" data-direction="right" src="/icons/chevron-right.svg"></img>
+                        </div>
+                    </div>
+                    <div class="current-year" data-quarter="1" data-year="2024">2024</div>
```

Comment:
@mdickson-adbe can we get current year programatically?

Reply:
Sure, that's no problem at all.